packages/suite/src/support/suite/ConnectedIntlProvider.tsx

```diff
@@ -1,13 +1,37 @@
-import * as React from 'react';
+import React, { useState, useEffect } from 'react';
 import { IntlProvider } from 'react-intl';
+import enMessages from '@trezor/suite-data/files/translations/en.json';
 import { useSelector } from '@suite-hooks/useSelector';
 import { isDev } from '@suite-utils/build';
+import type { Locale } from '@suite-config/languages';
+
+const useFetchMessages = (locale: Locale) => {
+    const [messages, setMessages] = useState<{ [key: string]: any }>({});
+
+    useEffect(() => {
+        let active = true;
+        const fetchMessages = async () => {
+            const messages =
+                locale === 'en'
+                    ? {}
+                    : await import(`@trezor/suite-data/files/translations/${locale}.json`)
```

Comment:
this dynamic import is not compatible with React Native, am I right? @Nodonisko

Reply:
I never heard of it and I can't even google it, so I think it should work. And if not, we can still create separate file `fetchMessages.ts` and `fetchMessages.native.ts` just for this function and it will be done.

Reply:
btw. i will be better if you will put this function outside hook and refactor use effect to something like this:
```ts
useEffect(() => {
  let active = true;
  fetchMessages(locale).then(result => {
    if (active) {
      setMessages({ ...enMessages, ...messages });
    }
  })
  return () => {
     active = false;
  };
...
```

it will be cleaner and easier to refactor in case this will be problem

Reply:
failing in CI https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1982995314#L343


old, so maybe no more relevant? https://dev.to/emilios1995/dynamic-imports-in-react-native-9k5

Reply:
Yea, now I can see what you mean. That probably will be the issue and I think we could solve this with solution that they used in link you send. I think if we will create object like this: 
const messages = {
 en: require(@trezor/suite-data/files/translations/en.json)
 cz: require(@trezor/suite-data/files/translations/cz.json)
}

then we can just access this object like `messages[locale]`. That should work both for web and native.

Reply:
let's address this outside this PR